### ISSUE TITLE
onDragStop not working  on img

### ISSUE THREAD

**Issue by candice2cc** [react-grid-layout/react-draggable#304]
code:
```
<Draggable>
   <img src={iconCircle}/>
</Draggable>
```

It works when modify the onMouseDown func as below:
```
  onMouseDown: EventHandler<MouseTouchEvent> = (e) => {
      e.preventDefault();
      dragEventFor = eventsFor.mouse; // on touchscreen laptops we could switch back to mouse

      return this.handleDragStart(e);
  };
```

**Comment by lucky-newbie** [comment]
it is solved in the 3.0.5 version？

**Comment by STRML** [comment]
See https://github.com/STRML/react-draggable/pull/98#issuecomment-147428973
